src/controllers/users.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const { UserService } = require('../services/userService');
+const userService = new UserService();
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getUserById(+userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (typeof name !== 'string' || !name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getUserById(+userId);
+
+  if (!foundUser || !userId) {
+    res.sendStatus(404);
+
+    return;
+  }
```

Comment:
have we a route on delete request /users/ - here we didn't transfer query or if userId is not passed it automatically transfer you on 404 and !userId check is not necessary?